source/plugins/music/index.mjs

```diff
@@ -13,7 +13,9 @@
       embed:/^\b$/,
     },
   }
-
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc.js';
+dayjs.extend(utc);
```

Comment:
Would it be ok to import **dayjs** and configure it in [source/app/metrics/utils](https://github.com/lowlighter/metrics/blob/master/source/app/metrics/utils.mjs#L10-L12) instead? 
That way other plugins will be able to use dayjs already configured without requiring additional imports (a bit like it's done for axios, puppetter, etc) 🙂

Reply:
I really don't know how but I'll take a look :)